tests/JustSaying.IntegrationTests/Fluent/AwsTools/WhenUsingABasicThrottle.cs

```diff
@@ -12,131 +8,131 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using NSubstitute;
-using Xunit;
-using Xunit.Abstractions;
+
 #pragma warning disable 618
 
-namespace JustSaying.IntegrationTests.Fluent.AwsTools
+namespace JustSaying.IntegrationTests.Fluent.AwsTools;
+
+public class WhenUsingABasicThrottle : IntegrationTestBase
 {
-    public class WhenUsingABasicThrottle : IntegrationTestBase
+    public WhenUsingABasicThrottle(ITestOutputHelper outputHelper)
+        : base(outputHelper)
     {
-        public WhenUsingABasicThrottle(ITestOutputHelper outputHelper)
-            : base(outputHelper)
-        {
-        }
+    }
 
-        protected override TimeSpan Timeout => TimeSpan.FromMinutes(5);
+    protected override TimeSpan Timeout => TimeSpan.FromMinutes(5);
 
-        [AwsTheory]
-        [InlineData(100)]
-        [InlineData(1000)]
-        public async Task Messages_Are_Throttled_But_Still_Delivered(int throttleMessageCount)
-        {
-            // Arrange
-            ILoggerFactory loggerFactory = OutputHelper.ToLoggerFactory();
-            IAwsClientFactory clientFactory = CreateClientFactory();
+    [AwsTheory]
+    [InlineData(100)]
+    [InlineData(1000)]
+#pragma warning disable xUnit1008
```

Comment:
I wonder if there's a bug here worth logging with xunit that derived classes aren't considered as part of the rule?